### PR TITLE
Works and about section

### PR DESCRIPTION
**Pull request by mertkantaroglu**
Hello,

I created works pages and about myself pages in this branch. I tried to make them as similar as I can to the original document.

However, on the about myself page, I could not solve the issues with the dev tool logos and their names. Please help me solve this issue.

Thanks,

### CODE REVIEW

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @mertkantaroglu, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project, that has a failed linter check. Kindly fix your linter errors and submit again 👍🏽 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -117,3 +113,202 @@ body {
...
+  order: 1;
+  flex-grow: 0;
+}
+
+.dev-tools {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+  padding-left: 10px;
```
Comment (emmiiorji) [inline styles.css]: Your design looks great. Kindly modify the list of languages to match the design template. The icons are in horizontal alignment with the names which have a background as shown in the illustration.

Your design:
image

Figma design:
image

File: styles.css [hunk 2]
```diff
@@ -117,3 +113,202 @@ body {
...
+
+  /* display: flex;
+  align-items: center;
+  justify-content: center; */
+}
```
Comment (emmiiorji) [inline styles.css]: Kindly remove unused pieces of code in order to have a more concise code base.

File: index.html [hunk 3]
```diff
@@ -76,5 +74,168 @@ <h3><a class="connect" href="#">LET'S CONNECT</a></h3>
...
+          </ul>
+          <p class="project-details">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="buttons">
+            <button type="button">html</button></li>
+            <button type="button">css</button>
+            <button type="button">javaScript</button>
+          </div>
```
Comment (emmiiorji) [inline index.html]: Considering that these technology listings are not buttons but a list, I recommend that you use `li` tags for each of the items and change the div tag to `ul` to enclose the `li` tags.

Remember to adjust this for each of the projects.

File: index.html [hunk 4]
```diff
@@ -76,5 +74,168 @@ <h3><a class="connect" href="#">LET'S CONNECT</a></h3>
...
+            class="picture"
+            src="Images/Snapshoot Portfolio (1).png"
+            alt="Snapshoot Portfolio Pic 2"
+          />
+          <h1 class="project-title">Multi-Post Stories</h1>
+          <ul class="project-summary">
+            <li><h3 class="canopy">CANOPY</h3></li>
+            <li><h3 class="back-end-dev">Back End Dev</h3></li>
+            <li><h3 class="year">2015</h3></li>
+          </ul>
```
Comment (emmiiorji) [inline index.html]: Kindly include the bullets for the stack and year details of the project as shown on the design.

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @mertkantaroglu, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @mertkantaroglu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_**Kindly check an optional comment I left under the review and work on it please**_ :pray: 

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [OPTIONAL] Kindly consider adding downward and forward arrows to your design under language section. Please take a look. :point_down: 

### Your Design
your-design


### Figma Design 
figma-design

**Pull request merged**